TECHNIX/iocBoot/iocTECHNIX-IOC-01/config.xml

```diff
@@ -9,8 +9,8 @@
 <macro name="PARITY" pattern="^(even)|(odd)|(none)$" description="Serial communication parity, defaults to none." defaultValue="none" hasDefault="YES" />
 <macro name="STOP" pattern="^[0-9]$" description="Serial communication stop bit, defaults to 1." defaultValue="1" hasDefault="YES" />
 
-<macro name="MAX_VOLT" pattern="^[0-9]+$" description="Maximum voltage" hasDefault="NO" />
-<macro name="MAX_CURR" pattern="^[0-9]+$" description="Maximum current" hasDefault="NO" />
+<macro name="MAX_VOLT" pattern="^[0-9]*\.?[0-9]*$" description="Maximum voltage in kV" hasDefault="NO" />
+<macro name="MAX_CURR" pattern="^[0-9]*\.?[0-9]*$" description="Maximum current in mA" hasDefault="NO" />
```

Comment:
perhaps we should add some sensible defaults for these - then we could remove doing so in the ioc test framework?

Reply:
I am worried about setting defaults for these, because if the actual device is rated lower than the default value and the default gets used, you might set unexpectedly high currents (it uses these values as scaling in the db records)...

Reply:
fair enough.